File: Buyer_BackEnd/Buyer_BackEnd/BuyerCartpage_temp/src/main/java/com/example/Dto/CartDTO.java
package com.example.dto;

import lombok.Data;

@Data
public class CartDto {

	private Long cartId;

	private Long seller_id;
	private Long productId;
	private String sellerEmailID;
	private String name;
	private String email;
	private String description;
	private String thumbnail;
	private double price;
	private int quantity;
	private String category;
	private String subcategory1;
	private String subcategory2;
	private Long totalproductPrice;

}

File: Buyer_FrontEnd/e-comm_UI_Buyer_SPRINT_5 3/e-comm_UI_Buyer_SPRINT_5/e-comm_UI_Buyer_5_SPRINT_5/src/app/cart-dto.ts
export class CartDto {
  constructor(
       public cartId: number,
       public seller_id : number,
      public productId: number,
      public sellerEmailID: string,
      public name: string,
      public email: string,
      public description: string,
      public thumbnail: string,
      public price: number,
      public quantity: number,
      public category: string,
      public subcategory1: string,
      public subcategory2: string,
      public totalproductPrice: number

    ) {}
}